source/extensions/config_subscription/grpc/xds_mux/grpc_mux_impl.cc

```diff
@@ -1,5 +1,6 @@
 #include "source/extensions/config_subscription/grpc/xds_mux/grpc_mux_impl.h"
 
+#include "envoy/config/endpoint/v3/endpoint.pb.h"
```

Comment:
This is the 3rd repetition of this logic, how come we have 3 mux implementations again? Was this the one old, one new, one unified? This seems really bad situation to do just about anything in xDS space :/